src/dal/src/infrastructure.js

```diff
@@ -0,0 +1,51 @@
+const https = require('https');
```

Comment:
Since we have to release, let's pass that, but afterwards please rework these copy&paste sections so that you don't have duplicate code.